ihatemoney/templates/project_feed.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0"
+	xmlns:dc="http://purl.org/dc/elements/1.1/"
+	xmlns:atom="http://www.w3.org/2005/Atom"
+	>
+    <channel>
+        <title>{{ g.project.name }}</title>
+        <description></description>
+        <atom:link href="{{ url_for(".feed", token=g.project.generate_token("feed"), _external=True) }}" rel="self" type="application/rss+xml" />
+        <link>{{ url_for(".list_bills", _external=True) }}</link>
+        {% for (weights, bill) in bills.items -%}
+        <item>
+            <title>{{ bill.what }} - {{ bill.amount|currency(bill.original_currency) }}</title>
+            <guid isPermaLink="false">{{ bill.id }}</guid>
+            <dc:creator>{{ bill.payer }}</dc:creator>
```

Comment:
[`<author>`](https://www.rssboard.org/rss-specification#ltauthorgtSubelementOfLtitemgt) needs to be an email address, but [`<dc:creator>`](https://www.rssboard.org/rss-profile#namespace-elements-dublin-creator) can be anything.